### PR TITLE
hw-mgmt: thermal: TC fix file reading issue

### PR DESCRIPTION
**Pull request by sholeksandr**
Fixed file reading issue in TC: "[Errno 19] No such device"

Bug #3534779 "hw-management-tc failed to start after cold reboot and no automatic restart"

Signed-off-by: Oleksandr Shamray <oleksandrs@nvidia.com>

### CODE REVIEW

File: usr/usr/bin/hw_management_thermal_control.py [hunk 1]
```diff
@@ -735,14 +735,12 @@ def get_file_val(self, filename, def_val=None, scale=1):
         @param scale: scale factor multiply with file value
         @return: int value
         """
-        val = None
+        val = def_val
         if self.check_file(filename):
             try:
                 val = int(self.read_file(filename)) / scale
-            except ValueError:
```
Comment (ciju-nvidia) [inline usr/usr/bin/hw_management_thermal_control.py:742]: Is this change intended for V.7.0030.1000_BR? or V.7.0030.2000_BR?

Comment (sholeksandr) [reply usr/usr/bin/hw_management_thermal_control.py:742]: V.7.0030.1000_BR

**Comment by sholeksandr** [comment]
RERUN

**Review by ciju-nvidia — APPROVED** [review]
